proxy/http/HttpConfig.cc

```diff
@@ -76,11 +76,12 @@ template <typename T> struct ConfigEnumPair {
 /// If found @a value is set to the corresponding value in @a list.
 template <typename T, unsigned N>
 static bool
-http_config_enum_search(const char *key, const ConfigEnumPair<T> (&list)[N], MgmtByte &value)
+http_config_enum_search(std::string_view key, const ConfigEnumPair<T> (&list)[N], MgmtByte &value)
 {
+  Debug("http_config", "enum element %s", key.data());
```

Comment:
Hmm seems a bit dangerous to assume a string_view passed as a param is null-terminated.  How about:
```
Debug("http_config", "enum element %.*s", static_cast<int>(key.size()), key.data());
```